src/containers/App/App.js

```diff
@@ -2,7 +2,12 @@ import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
 import { IndexLink } from 'react-router';
 import { LinkContainer } from 'react-router-bootstrap';
-import Navbar from 'react-bootstrap/lib/Navbar';
```

Comment:
As far as I have seen it, the `Namespace.Component` behavior isn't supported by `transform-react-inline-elements`.